trpc/router/goal.ts

```diff
@@ -215,6 +216,11 @@ export const goal = router({
                 project: goal.project ? addCalculatedProjectFields(goal.project, ctx.session.user.activityId) : null,
                 estimate: getEstimateListFormJoin(goal),
                 activityFeed: mixHistoryWithComments(history, goal.comments),
+                relations: makeGoalRelationMap({
```

Comment:
Can you open issue about using underscore for all calculated fields? It helps to understand on client what type of data we have. Thanks 🙏🏼

Reply:
#1263